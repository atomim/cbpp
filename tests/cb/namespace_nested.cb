x# = 25.0
y# = 35.0
zoom# = 1.0
dim sx as float
dim sy as float

namespace project <x,y,z,sx,sy,zoom>
	dim mult as float
	
	namespace calc_mult<mult, zoom>
		temp# = zoom * 1.5
		mult = temp
	endnamespace
	
	sx = (mult * x)/z
	sy = (mult * y)/z
endnamespace

dot sx,sy