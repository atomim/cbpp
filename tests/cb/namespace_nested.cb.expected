x# = 25.0
y# = 35.0
zoom# = 1.0
dim sx as float
dim sy as float

// namespace project <x,y,z,sx,sy,zoom>
	dim _project_mult as float
	
	// namespace calc_mult<mult, zoom>
		_calc_mult_project_temp# = zoom * 1.5
		_project_mult = _calc_mult_project_temp
	// endnamespace
	
	sx = (_project_mult * x)/z
	sy = (_project_mult * y)/z
// endnamespace

dot sx,sy